include/beman/dump/dump.hpp

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef BEMAN_DUMP_DUMP_HPP_INCLUDED
+#define BEMAN_DUMP_DUMP_HPP_INCLUDED
+
+#include <array>
+#include <print>
+#include <string_view>
+
+namespace beman::dump {
+    namespace detail {
+        template <std::size_t N>
+        constexpr auto format_string = []{
+            std::array<char, N * 3 - 1> result{};
+
+            auto ptr = &result[0];
+            for (auto i = 0uz; i != N; ++i)
+            {
+                if (i > 0)
+                    *ptr++ = ' ';
+                *ptr++ = '{';
+                *ptr++ = '}';
+            }
+
+            return result;
+        }();
```

Comment:
A minor suggestion for improving style:

```c++
constexpr auto format_string = std::invoke([] { /*...*/ } );
```

Reply:
Making the immediate call to a lambda visible is very reasonable!

Reply:
Done